engine/src/hopeit/app/config.py

```diff
@@ -244,9 +251,13 @@ class EventConfig:
     response_timeout: float = 60.0
     logging: EventLoggingConfig = field(default_factory=EventLoggingConfig)
     stream: EventStreamConfig = field(default_factory=EventStreamConfig)
+    extras: Dict[str, Any] = field(default_factory=dict)
 
+    def get(self, *, key: str = '_', datatype: Type[EventSettingsType] = Any) -> EventSettingsType:
```

Comment:
datatype=Any does not work. It should be None by default and return a Dict[str, Any] without parsing in that case.